scripts/firedrake-install

```diff
@@ -790,6 +663,33 @@ def pip_uninstall(package):
             raise InstallError("pip claims it uninstalled %s more than 10 times.  Something is probably broken.", package)
 
 
+def build_and_install_petsc():
+    import hashlib
+    tarball = "eigen-3.3.3.tgz"
+    url = "http://bitbucket.org/eigen/eigen/get/3.3.3.tar.gz"
```

Comment:
I think that we will want to rewrite this stuff in the future.  But note that if I just download a newer version (by changing the url, tarball name and expected hash) then all that happens is I leave the old guy around.